akka-docs/src/main/paradox/typed/cluster-sharded-daemon-process.md

```diff
@@ -0,0 +1,58 @@
+# Sharded Daemon Process
+
+
+@@@ warning
+
+This module is currently marked as @ref:[may change](../common/may-change.md) because it is a new feature that
+needs feedback from real usage before finalizing the API. This means that API or semantics can change without
+warning or deprecation period. It is also not recommended to use this module in production just yet.
```

Comment:
Good with ApiMayChange. I think it could be ready for production if we like. Can't do much harm.